etc/utils.js

```diff
@@ -176,4 +176,20 @@ if (m > 6) {
 	}
 }
 
-exports.sponsorsScope = sponsorsScope;
\ No newline at end of file
+exports.sponsorsScope = sponsorsScope;
+
+exports.makeUserPaid = function(req, res, nextSteps) {
```

Comment:
Is there a reason this is in `utils.js`? Also, you're using this function right? For some reason I can't seem to find it when I search for it on this page..